doc/orch: How to update an existing specification (#35099)
ceph/ceph: doc/mgr/orchestrator.rst

        - host2
        - host3

Updating Service Specifications
===============================

> The Ceph orchestrators store a declarative state of the
> specifications. For certain operations, like updating

Reviewer: ```suggestion
The Ceph Orchestrator maintains a declarative state of each
service in a ``ServiceSpec``. For certain operations, like updating
```

the RGW HTTP port, we need to update the existing
specification.

> First, get the current configuration::

Reviewer: ```suggestion
First, get the current ``ServiceSpec``::
```

> First, get the current ``ServiceSpec``::

Reviewer 2: In this instance `get` means to list? yes? If so, please say "list" instead of get.

Reviewer 2: ```suggestion
1. List the current ``ServiceSpec``::

Author: fixed

>     ceph orch --service_name=<service-name> --export > myservice.yaml

Reviewer: ```suggestion
    ceph orch ls --service_name=<service-name> --export > myservice.yaml
```

> Now, update the yaml file (like changing the port)::

Reviewer 2: ```suggestion
2. Update the yaml file (like changing the port)::

Reviewer 2: What does "(like changing the port)" mean?

Author: like changing the `spec.rgw_frontend_port` property of the spec.

Reviewer 2: I think maybe make that clear and expand that if you're  going to include "like changing the port" or link to another section that details these instructions? It looks a little out of place.

    vi myservice.yaml

> Finally, apply the new spec::

Reviewer: ```suggestion
Finally, apply the new ``ServiceSpec``::
```

> Finally, apply the new ``ServiceSpec``::

Reviewer 2: ```suggestion
3. Apply the new ``ServiceSpec``::

Author: fixed
